### PR TITLE
Add sidekiq and configure queue adapter

### PR DESCRIPTION
**Pull request by ericaraujo13**
Closes: <https://github.com/comarev/comarev/issues/67>

References: <issue-link>/<pr-link>

#### What?

- What this PR is changing in the code.
- This Pr adds a queue adapter and sidekiq to the project.

#### Why?

- Why we are performing these changes.
- To be able to perform future jobs with a nice performance

#### How to test it?

- Describe how can we test the new feature/fixed bug.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,6 +17,8 @@ gem 'puma', '~> 4.1'
 gem 'pundit'
 gem 'rack-cors'
 gem 'rails', '~> 6.0.3', '>= 6.0.3.5'
+gem 'sidekiq'
+gem 'redis'
```
Comment (ronaldoaraujo) [inline Gemfile:21]: I think it's not necessary. Redis is already included in Sidekiq dependency.

**Review by m-pereira — COMMENTED** [review]
Very nice job 🚀 
Can you please remove de `redis` gem from gemfile? Then I think it's ready to be merged 😄

**Comment by m-pereira** [comment]
It's easier than I thought 🙌🏻 

```
[Correctable] Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem redis should appear before sidekiq.
```

**Review by m-pereira — APPROVED** [review]

**Comment by m-pereira** [comment]
@allcontributors add @ericaraujo13 for code and @ronaldoaraujo for review

**Comment by m-pereira** [comment]
@allcontributors add @ronaldoaraujo for review

**Pull request merged**